src/summary.rs

```diff
@@ -272,6 +287,22 @@ impl Message {
 
         summary.split_whitespace().collect::<Vec<&str>>().join(" ")
     }
+
+    async fn get_vcard_summary(&self, context: &Context) -> Result<Option<String>> {
+        ensure!(self.viewtype == Viewtype::Vcard, "Message is not a vCard");
+        let Some(path) = self.get_file(context) else {
+            bail!("No file");
+        };
+        let vcard = tokio::fs::read(path)
```

Comment:
This should be optimised somehow probably, it's not good that the file is read every time the chatlist is built

Reply:
If we add a new stock string anyway, we can just always use it. I.e. simply always write "👤 Virtual contact file" (or "👤 Contact", users don't know what a virtual contact file is) into the chatlist. This way, we keep things simpler.

Reply:
I just tried to do what WhatsApp (and probably others) does, it displays the contact name. Would  be nice if we can do the same. E.g. parse a vCard upon a receipt and save the contact name to the db. Btw, while we don't allow to attach a custom text to vCard messages, we can attach the contact name on the sender side as a temporary solution

Reply:
Btw, i added a new stock string just in case (and for uniformity with other viewtypes) because if we display the file name as a fallback, the stock string should never be displayed

Reply:
A new stock string is some work for all the UIs to set it and for translators, so we should add it only with a good reason. I renamed it to "Contact" now which already is a translated string in the UIs (it's just not passed to core), so it's just the UIs that have to set a new stock string and way less work for everyone.

People won't send contacts _that_ often, when they do, it sometimes won't be the last message (that is shown in the chatlist) and when it is, it won't make a big difference whether we write `Contact` or the contact's name. So, let's just call it good enough for now in order to make progress :)